decentraland/package.json

```diff
@@ -5,9 +5,13 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "scripts": {
-    "build:android": "npx expo build:android",
     "start": "FORCE_COLOR=1 node starter.js",
+    "build:android": "npx expo build:android",
     "build:ios": "npx expo build:ios",
+    "build:dev:android": "npx expo build:android --release-channel dev",
+    "build:dev:ios": "npx expo build:ios --release-channel dev",
+    "build:stag:android": "npx expo build:android --release-channel stag",
+    "build:stag:ios": "npx expo build:ios --release-channel stag",
```

Comment:
I'm not sure if we'll use both the dev and stag channels just yet, but I suppose it can't hurt to have both of them configured now

Reply:
I've thought to create only the `dev` script, but I created `stag` too because I've thought that you could want it too.

Reply:
Yep, makes sense to me. Resolving this!